WordPress/src/main/java/org/wordpress/android/ui/main/SitePickerActivity.java

```diff
@@ -136,14 +136,6 @@ protected void onSaveInstanceState(Bundle outState) {
         super.onSaveInstanceState(outState);
     }
 
-    @Override
-    public void finish() {
-        super.finish();
-        if (mDidUserSelectSite) {
```

Comment:
It looks like `mDidUserSelectSite` isn't used anywhere else in the code.  We can remove the variable completely.